main.go

```diff
@@ -24,6 +25,11 @@ import (
 	"github.com/batchcorp/plumber/prometheus"
 )
 
+const (
+	// Note: Only used if analytics are enabled (default: no)
+	MixPanelToken = "2d2995d51084155edc043905f2dc528f"
```

Comment:
Should probably be injected via build tags

Reply:
👍